### PR TITLE
Add new Anomaly Detector demo

### PR DESCRIPTION
**Pull request by AlbertDavletov**

### CODE REVIEW

File: Documentation/AnomalyDetectorDemo.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Anomaly Detector
+
+![alt text](https://github.com/Microsoft/Cognitive-Samples-IntelligentKiosk/blob/master/Documentation/AnomalyDetectorDemo.png "Anomaly Detector")
+
+This demo shows anomaly detection using the Anomaly Detector API. There are many types of time-series data—and no one algorithm fits them all. Anomaly Detector looks at your time-series data set and automatically selects the best algorithm from the model gallery to ensure high accuracy for your specific scenario: business incidents, monitoring IoT device traffic, managing fraud, responding to changing markets and more.
```
Comment (CarlosP-MS) [inline Documentation/AnomalyDetectorDemo.md]: This demo shows anomaly detection using the Anomaly Detector API. There are a three built-in scenarios that you can use out of the box using pre-selected data streams and story lines, as well as a Live Sound scenario that samples the audio level of your microphone as the input data stream.

File: Documentation/AnomalyDetectorDemo.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# Key Source Code
+
+* [AnomalyDetectorDemo](../Kiosk/Views/AnomalyDetector/AnomalyDetectorDemo.xaml.cs): Main page that drives the demo. It hosts the AnomalyChartControl (see below) to display the chart, setup a microphone to capture and process audio data. It then uses the result to detect anomaly data.
+
+* [AnomalyChartControl](../Kiosk/Views/AnomalyDetector/AnomalyChartControl.xaml.cs): The code that contains the chart and enables different type of scenarios.
+
+* [ScenarioInfoControl](../Kiosk/Views/AnomalyDetector/ScenarioInfoControl.xaml.cs): The code that contains the scenario display information.
```
Comment (CarlosP-MS) [inline Documentation/AnomalyDetectorDemo.md]: Displays the story line behind each scenario.

File: Documentation/AnomalyDetectorDemo.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+![alt text](https://github.com/Microsoft/Cognitive-Samples-IntelligentKiosk/blob/master/Documentation/AnomalyDetectorDemo.png "Anomaly Detector")
+
+This demo shows anomaly detection using the Anomaly Detector API. There are many types of time-series data—and no one algorithm fits them all. Anomaly Detector looks at your time-series data set and automatically selects the best algorithm from the model gallery to ensure high accuracy for your specific scenario: business incidents, monitoring IoT device traffic, managing fraud, responding to changing markets and more.
+
+# Key Source Code
+
+* [AnomalyDetectorDemo](../Kiosk/Views/AnomalyDetector/AnomalyDetectorDemo.xaml.cs): Main page that drives the demo. It hosts the AnomalyChartControl (see below) to display the chart, setup a microphone to capture and process audio data. It then uses the result to detect anomaly data.
+
+* [AnomalyChartControl](../Kiosk/Views/AnomalyDetector/AnomalyChartControl.xaml.cs): The code that contains the chart and enables different type of scenarios.
```
Comment (CarlosP-MS) [inline Documentation/AnomalyDetectorDemo.md]: ... types of scenarios.

File: Documentation/AnomalyDetectorDemo.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Anomaly Detector
+
+![alt text](https://github.com/Microsoft/Cognitive-Samples-IntelligentKiosk/blob/master/Documentation/AnomalyDetectorDemo.png "Anomaly Detector")
+
+This demo shows anomaly detection using the Anomaly Detector API. There are many types of time-series data—and no one algorithm fits them all. Anomaly Detector looks at your time-series data set and automatically selects the best algorithm from the model gallery to ensure high accuracy for your specific scenario: business incidents, monitoring IoT device traffic, managing fraud, responding to changing markets and more.
+
+# Key Source Code
+
+* [AnomalyDetectorDemo](../Kiosk/Views/AnomalyDetector/AnomalyDetectorDemo.xaml.cs): Main page that drives the demo. It hosts the AnomalyChartControl (see below) to display the chart, setup a microphone to capture and process audio data. It then uses the result to detect anomaly data.
```
Comment (CarlosP-MS) [inline Documentation/AnomalyDetectorDemo.md]: Main page that drives the demo. It hosts the AnomalyChartControl (see below) to display the API results, sets up all the scenarios, including the initialization/handling of the audio processing pipeline for the Live Sound scenario.

File: Kiosk/Views/AnomalyDetector/AnomalyDetectionScenarioData.cs [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+        yearly,
+    }
+
+    public class AnomalyDetectionScenario
+    {
+        public const int DefaultRequiredPoints = 13;
+
+        public AnomalyDetectionScenarioType ScenarioType { get; set; }
+        public string FilePath { get; set; }
+        public GranType Granuarity { get; set; }
```
Comment (CarlosP-MS) [inline Kiosk/Views/AnomalyDetector/AnomalyDetectionScenarioData.cs]: typo

File: README.md [hunk 6]
```diff
@@ -20,6 +20,7 @@ The Intelligent Kiosk Sample is a collection of demos showcasing workflows and e
 
 | Scenario                     | Overview | Features Covered  |
 | ---------------------------- | -------- | ----------------  |
+| [Anomaly Detector](Documentation/AnomalyDetectorDemo.md)      | A realtime workflow for processing data from a microphone to show point anomalies exported by Microsoft Anomaly Detector Service. | Anomaly Detector API |
```
Comment (CarlosP-MS) [inline README.md]: Showcases anomaly detection with three pre-selected data streams, as well as a Live Sound scenario that samples the audio level of your microphone as the input data stream.

**Review by CarlosP-MS — CHANGES_REQUESTED** [review]

File: Documentation/AnomalyDetectorDemo.md (hunk 1 above)
Comment (AlbertDavletov) [reply Documentation/AnomalyDetectorDemo.md]: Fixed

File: Documentation/AnomalyDetectorDemo.md (hunk 2 above)
Comment (AlbertDavletov) [reply Documentation/AnomalyDetectorDemo.md]: Fixed

File: Documentation/AnomalyDetectorDemo.md (hunk 3 above)
Comment (AlbertDavletov) [reply Documentation/AnomalyDetectorDemo.md]: Fixed

File: Documentation/AnomalyDetectorDemo.md (hunk 4 above)
Comment (AlbertDavletov) [reply Documentation/AnomalyDetectorDemo.md]: Fixed

File: Kiosk/Views/AnomalyDetector/AnomalyDetectionScenarioData.cs (hunk 5 above)
Comment (AlbertDavletov) [reply Kiosk/Views/AnomalyDetector/AnomalyDetectionScenarioData.cs]: Fixed

File: README.md (hunk 6 above)
Comment (AlbertDavletov) [reply README.md]: Fixed

**Review by CarlosP-MS — APPROVED** [review]

**Pull request merged**